firebase-perf/src/main/java/com/google/firebase/perf/gauges/CpuGaugeCollector.java

```diff
@@ -96,6 +97,7 @@ private CpuGaugeCollector() {
   }
 
   @VisibleForTesting
+  @Inject
```

Comment:
Since cpuMetricReadings is always initialized to the same thing, why not initialize it where it's declared? It doesn't actually depend on the constructor's parameters.

Reply:
Good catch! Done.